### PR TITLE
Fix validator

### PR DESCRIPTION
**Pull request by ZevEisenberg**
This has some breaking changes with the validator, but I don't think anyone was using it.

### CODE REVIEW

File: Pod/Classes/Forms/FieldValidator.swift [hunk 1]
```diff
@@ -8,38 +8,14 @@
...
-            validationErrors = failedErrors
-        }
-        return validationErrors
-    }
+    let field: String
+    let value: String
+    public let validationErrors: [Error]
 
     public var localizedDescription: String {
+        print(validationErrors.first!.localizedDescription)
```
Comment (ateliercw) [inline Pod/Classes/Forms/FieldValidator.swift]: #req remove this line

File: Pod/Classes/Forms/FieldValidator.swift [hunk 2]
```diff
@@ -72,21 +53,28 @@ public struct FieldValidator {
...
-            throw RuleErrors.shouldNotBeEmpty
+            throw RuleError.empty
         }
     }
 
 }
 
 public struct LengthValidator: FieldValidationRule {
 
+    public init() {
```
Comment (ateliercw) [inline Pod/Classes/Forms/FieldValidator.swift]: #pp mark as unavailable

**Comment by ZevEisenberg** [comment]
Holding off on merge while @nevillco and I take this through its paces on an internal project, in case we find other tweaks we want to make.

**Comment by ZevEisenberg** [comment]
@ateliercw we're good to go on this. The stuff that's in this PR has all been tested, at least provisionally, in a project. Feel free to re-check and merge any time if you're good to go.

**Pull request merged**